docs/source/user_documentation/user_documentation.rst

```diff
@@ -125,6 +125,21 @@ SnowShu will report details of the created replica once completed.
 
 .. image:: /../assets/completed_replica.png 
 
+Creating An Incremental Replica
+------------------
+
+You have already built a replica but made some changes in ``replica.yml`` file. Instead of building a new replica, you can apply changes to the existing replica.
+Incremental replica creates relations and loads data only for new entries found in ``replica.yml`` file, which are not already present in target replica image.
```

Comment:
Maybe make a statement that target for incremental is actually a docker image name, which you can find by typing: docker images, and that it's usually starts with snowshu_replica_ prefix